### PR TITLE
[GP-4] fix typos

### PR DESCRIPTION
**Pull request by akkspros**
fix typos

[GP-4]

[GP-4]: https://moderntribe.atlassian.net/browse/GP-4

### CODE REVIEW

**Comment by jrserani** [comment]
_Ref:_ [#GTRIA-108](https://moderntribe.atlassian.net/browse/GTRIA-108)

Thank you for submitting this - we're taking a look and will let you know if we have any questions or comments :-)

File: admin/db.php [hunk 1]
```diff
@@ -314,7 +314,7 @@ function gigpress_db_upgrade_160() {
 		}
 	}
 
-	// Try our darndest to extract states from cities and put them in their own column
+	// Try our darnedest to extract states from cities and put them in their own column
```
Comment (cliffordp) [inline admin/db.php]: this isn't a typo
image

However, we could use less colloquial wording, such as "Try our hardest..."

Comment (sc0ttkclark) [reply admin/db.php]:
```suggestion
	// Try our hardest to extract states from cities and put them in their own column
```

File: readme.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 === GigPress ===
-Contributors: ModernTribe, brianjessee, camwynsp, sc0ttkclark, aguseo, barry.hughes, bordoni, borkweb, brook-tribe, cliffpaulick, courane01, GeoffBel, geoffgraham, ggwicz, leahkoerper, neillmcshea, nicosantos, peterchester, reid.peifer, shane.pearlman, shelbelliott, tribecari, vicskf, zbtirrell
+Contributors: ModernTribe, brianjessee, camwynsp, sc0ttkclark, aguseo, barry.hughes, bordoni, borkweb, brook-tribe, cliffpaulick, courane01, GeoffBel, geoffgraham, ggwicz, leahkoerper, neillmcshea, nicosantos, peterchester, reid.peifer, shane.pearlman, shelbelliott, tribecari, vicskf, zbtirrell, passoniate
```
Comment (sc0ttkclark) [inline readme.txt]:
```suggestion
Contributors: ModernTribe, brianjessee, camwynsp, sc0ttkclark, aguseo, barry.hughes, bordoni, borkweb, brook-tribe, cliffpaulick, courane01, GeoffBel, geoffgraham, ggwicz, leahkoerper, neillmcshea, nicosantos, peterchester, reid.peifer, shane.pearlman, shelbelliott, tribecari, vicskf, zbtirrell
```

**Comment by cliffordp** [comment]
Scott's work looks good.

**Pull request merged**